pkg/rpcclient/nep11/base.go

```diff
@@ -246,3 +247,71 @@ func UnwrapKnownProperties(m *stackitem.Map, err error) (map[string]string, erro
 	}
 	return res, nil
 }
+
+// TransferEventsFromApplicationLog retrieves all emitted TransferEvents from the
+// provided ApplicationLog.
+func TransferEventsFromApplicationLog(log *result.ApplicationLog) ([]*TransferEvent, error) {
```

Comment:
@notimetoname would probably be interested to see this and imagine what it may look like if we're to use generics.

Reply:
We can make use of generics here as far, https://github.com/nspcc-dev/neo-go/issues/2377#issuecomment-1563185046.